test/testenv.py

```diff
@@ -49,8 +50,10 @@ def get_clang_args(self):
 
         if self.domain == 'c':
             clang_args.extend(getattr(conf, 'hawkmoth_clang_c', []))
+            clang_args.extend(compiler.get_include_args('clang', 'c'))
         else:
             clang_args.extend(getattr(conf, 'hawkmoth_clang_cpp', []))
+            clang_args.extend(compiler.get_include_args('clang', 'c++'))
```

Comment:
These two should get the compiler config from conf.py. We already import it for other configs and use e.g. `clang_args = getattr(conf, 'hawkmoth_clang', [])`.

For #254 I imagine we'll need to somehow pass the compiler path from the workflow yaml. It's a bit easier if we can handle that in conf.py alone, not everywhere.

Reply:
The order in which the final args are added should be the same in both the extension and here.